src/components/Person/Person.jsx

```diff
@@ -1 +1,9 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {
+      person.age && <p className="Person__age">{person.age && `I am ${person.age}`}</p>
+    }
+    <p className="Person__partner">{!person.isMarried ? 'I am not married' : `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}</p>
```

Comment:
This link looks too long and too complicated, let's create a separate variable for partner and use it here, also try to avoid nested ternary

`const partner = person.sex === 'm' ? 'wife' : 'husband';`

Reply:
Thank you for advice!